root/components/EntityTabs.js

```diff
@@ -124,6 +124,21 @@ function buildLinks(
     ));
   }
 
+  if (entity.entityType === 'event') {
```

Comment:
We could, though adding support for series would necessitate more large amounts of refactoring, since we assume each archive only stores one entity type. :) But yes, it can't hurt.